tests/Flash/FlashTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Sitchco\Tests\Flash;
+
+use Sitchco\Flash\AdminNotificationService;
+use Sitchco\Flash\Flash;
+use Sitchco\Tests\Support\TestCase;
+
+/**
+ * Class FlashTest
+ * @package Sitchco\Tests
+ */
+class FlashTest extends TestCase
+{
+    private Flash $flash;
+    private AdminNotificationService $service;
+
+    /**
+     * Set up the Flash service for testing.
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->service = new AdminNotificationService();
+        $this->flash = new Flash($this->service);
+    }
+
+    /**
+     * Test that notifications are properly rendered and cleared.
+     */
+    public function testRender(): void
+    {
+        $this->service->dispatch("Test notification");
+        $this->service->dispatchError("Error notification");
+        $this->flash->shutdown();
+
+        ob_start();
+        $notifications = $this->flash->render();
+        $output = ob_get_clean();
+
+        // Ensure the notifications are displayed
+        $this->assertStringContainsString("Test notification", $output);
+        $this->assertStringContainsString("Error notification", $output);
+        $this->assertCount(2, $notifications);
+
+        // Ensure notifications are cleared after rendering
+        $this->assertFalse($this->service->hasNotifications());
+    }
+
+    /**
+     * Test that notifications are stored and retrieved properly.
+     */
+    public function testShutdownStoresNotifications(): void
```

Comment:
I don't think this test is needed. Doesn't it test the same thing as the previous test?